### PR TITLE
doc: fix arguments for install, upgrade and remove

### PR DESCRIPTION
**Pull request by kontura**
For: https://github.com/rpm-software-management/dnf5/issues/1602
For: https://github.com/rpm-software-management/dnf5/issues/1498

Also adds some clarification to `group_package_types` docs.

### CODE REVIEW

File: doc/commands/install.8.rst [hunk 1]
```diff
@@ -25,15 +25,23 @@
...
 Description
 ===========
 
-The ``install`` command in ``DNF5`` is used for installing packages. It makes sure that
-all given packages defined in ``package-spec`` arguments and their dependencies are installed
-on the system. Environments can be installed with ``@environment-id`` as ``<package-spec>``.
+The ``install`` command in ``DNF5`` is used for installing packages, groups or environments.
+
+When installing packages defined in ``package-spec`` arguments, ``DNF5`` ensures that the packages
+and their dependencies are installed on the system.
```
Comment (jan-kolarik) [inline doc/commands/install.8.rst:37]: > ensures that the packages and their dependencies are installed on the system.

Hmm, it seems we are still promising to do something that we aren't actually doing, e.g. what's described in https://github.com/rpm-software-management/dnf5/issues/1498 ...

Comment (jan-kolarik) [reply doc/commands/install.8.rst:37]: Consider adding something like: "If a specified package is already installed, DNF5 does not check its dependencies again and simply verifies that the package itself is present."

Comment (jan-kolarik) [reply doc/commands/install.8.rst:37]: If the described situation also occurs for group and env specs, I’d suggest adding it as a general note under the first sentence of the description.

File: doc/misc/specs.7.rst [hunk 2]
```diff
@@ -161,12 +161,12 @@ Specifying File Provides
...
 
 
-Groups
+Comps
 ======
 
-``<group-spec>`` allows one to select (environment) groups a particular operation should work
-on. It is a case insensitive string (supporting globbing characters) that is
-matched against a group's ID, canonical name and name translated into the
+``<group-spec>``|``<environment-spec>`` allows one to select groups|environments a particular
```
Comment (jan-kolarik) [inline doc/misc/specs.7.rst:167]: Perhaps mention that commands accepting multiple types of specs uses the `@` prefix?

File: doc/commands/install.8.rst (hunk 1 above)
Comment (kontura) [reply doc/commands/install.8.rst:37]: I tried to phrase it in such a way that specifies we check the dependencies only when actually installing the package so I think it should cover the mentioned issue.

However I can add the sentence to make it explicit.

Comment (kontura) [reply doc/commands/install.8.rst:37]: > If the described situation also occurs for group and env specs, I’d suggest adding it as a general note under the first sentence of the description.

It doesn't, both groups and environments can actually be installed (synchronized) over and over again and each time the dependencies/content is processed.

I have added the note specific for packages.

File: doc/misc/specs.7.rst (hunk 2 above)
Comment (kontura) [reply doc/misc/specs.7.rst:167]: Added.

File: doc/commands/install.8.rst (hunk 1 above)
Comment (jan-kolarik) [reply doc/commands/install.8.rst:37]: > However I can add the sentence to make it explicit.

Thanks, it's now more understandable and explicit.

**Review by jan-kolarik — APPROVED** [review]
Thanks a lot for improving the docs!

**Pull request merged**